### DISCUSSION TITLE
closed issues ?

### DISCUSSION THREAD

**Discussion by vturpin** [OceanGlidersCommunity/OG-format-user-manual#6] [Q&A] [answered]
Do we loose the issues when they are closed ? Or can we find them somewhere ?

**Comment by soerenthomsen** [comment, marked as answer]
<img width="388" alt="Screen Shot 2021-09-10 at 12 29 00" src="https://user-images.githubusercontent.com/32633545/132840410-b554bc3f-f249-47fb-8a71-77e7987510d9.png">

we keep everything!
